lbm_lib/smtc_modem_core/smtc_ral/src/ral_sx128x.c

```diff
@@ -292,6 +303,7 @@ ral_status_t ral_sx128x_cfg_rx_boosted( const void* context, const bool enable_b
 ral_status_t ral_sx128x_set_rx_tx_fallback_mode( const void* context, const ral_fallback_modes_t ral_fallback_mode )
 {
     bool fallback_mode_is_fs;
+    /* EvaTODO: what to do with front-end module? */
```

Comment:
Maybe add a `LOG_WRN` here that the frontend is not doing anything in this function.